include/wx/msw/ole/oleutils.h

```diff
@@ -66,18 +66,45 @@ inline void wxOleUninitialize()
 class WXDLLIMPEXP_CORE wxBasicString
 {
 public:
-    // ctors & dtor
-    wxBasicString(const wxString& str);
-    wxBasicString(const wxBasicString& bstr);
-    ~wxBasicString();
 
-    wxBasicString& operator=(const wxBasicString& bstr);
-
-    // accessors
-        // just get the string
+	// Takes over the ownership of bstr
+    wxBasicString(BSTR bstr = NULL);
```

Comment:
I can change it but TBH I do not see what situations it could prevent. 

BTW, `wxBSTR` I submitted before had `wxBSTR(BSTR bstr, bool copy)` ctor, modelled after `_bstr_t`'s ctor.